src/simd.h

```diff
@@ -60,10 +60,13 @@ namespace simd {
 #endif
 
 // Include basic operations
-#include "simd_basic.h"
+#include "simd_basic_fnt.h"
```

Comment:
Comment above should be updated to `Include FNT operations.` or `Include accelerated operations for FNT.`